src/components/Photos/index.js

```diff
@@ -11,12 +11,149 @@ import { wallpaperPhotos } from './imgCategories/Wallpapers';
 
 import './Photos.css';
 
+const categories = {
+    "All": "",
+    "StuySquad": stuysquadPhotos,
+    "Sing": singPhotos,
+    "SOS": sosPhotos,
+    "Club": clubPhotos,
+    "Multigen": multigenPhotos,
+    "Wallpapers": wallpaperPhotos,
+    "Logos": logoPhotos,
+    "Merch": merchPhotos,
+    "Others": otherPhotos,
+};
+
+const sorts = [ "Date Asc.", "Date Desc." ];
+
+const get_photos = (category,sort) => {
+    if (category === "All") {
+	var c = Object.keys(categories)
+	c.shift()
+	var dates = []
+	c.map(category => (
```

Comment:
Speaking of `let`. I think you can simplify the code more if you were to just have the if statement set a variable called `dates` depending on if the category is all or single category. Then you can run sort and return on that one variable instead of repeating the same code twice.

Reply:
I'm not sure how to do this